### PR TITLE
hmcts/rpx-xui-node-lib@2.25.1

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3547

### Change description ###

Fix for the 403 and 401 errors

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: package.json
Comment (paul-graham) [inline package.json:77]: What happened to `2.25.1`? 🤔

**Comment by paul-graham** [comment]
Also, where is the PR for the node lib that this references? Has that already been approved and merged?

File: package.json
Comment (udaydenduluri33) [reply package.json:77]: 2.25.1 does not have logging. So had to create one additional version

**Comment by udaydenduluri33** [comment]
> Also, where is the PR for the node lib that this references? Has that already been approved and merged?

yes please its already merged. Thanks

**Comment by udaydenduluri33** [comment]
> > Also, where is the PR for the node lib that this references? Has that already been approved and merged?
> 
> yes please its already merged. Both Prs are linked to the JIRA. Thanks

**Review by paul-graham — APPROVED** [review]
Been back and looked at the PRs for `rpx-xui-node-lib` and they look okay, but they shouldn't have been merged without sufficient approvals.

I think the configuration on GitHub needs to be updated to only allow a merge into `master` after at least 2 approvals on that project.

**Comment by udaydenduluri33** [comment]
> Been back and looked at the PRs for `rpx-xui-node-lib` and they look okay, but they shouldn't have been merged without sufficient approvals.
> 
> I think the configuration on GitHub needs to be updated to only allow a merge into `master` after at least 2 approvals on that project.

Yes agree

**Pull request merged**
